.github/workflows/agent-tox.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: [self-hosted, linux, jammy, X64]
     strategy:
       matrix:
-        python: ["3.8", "3.10"]
+        python: ["3.8.12", "3.10"]
```

Comment:
I think this was supposed to be 3.8.10 to match the version we're targeting for now right?
From focal:
```
$ python3 --version
Python 3.8.10
```

Reply:
If we make that `3.8.10` we'll get an error because that version is not available on `jammy`. `3.8.12` is the lowest version available on `jammy` _and_ it's still below `3.8.17`, where `filter` became available. So by setting that to `3.8.12`, even though we don't match the version on the agent, we still manage to test on a version that doesn't support filtering (i.e. testing the patch), without going into all the trouble of setting up `pyenv`.